src/v2/Apps/Purchase/Utils/orderHelper.tsx

```diff
@@ -0,0 +1,36 @@
+import {
+  CheckCircleFillIcon,
+  PendingCircleIcon,
+  XCircleIcon,
+} from "@artsy/palette"
+import React from "react"
+
+export const getOrderIcon = status => {
+  switch (status) {
+    case "submitted":
+      return <PendingCircleIcon fill="black60" />
+    case "canceled":
+      return <XCircleIcon fill="red100" />
+    case "refunded":
+      return <XCircleIcon fill="red100" />
+    case "confirmed":
```

Comment:
Didn't trace the code but is it "confirmed" or "approved"?

Reply:
😬 Yes it [is](https://github.com/artsy/metaphysics/pull/2762) `APPROVED`  🙌

Reply:
fixed in https://github.com/artsy/force/pull/6436/commits/264aa529f9819d14d17d2edd2366a0fd2cb7b33d